sync/legacy.py

```diff
@@ -0,0 +1,63 @@
+from collections import defaultdict
+from dataclasses import dataclass
+from typing import Sequence, Dict
+
+import requests
+import tarfile
+from io import BytesIO
+
+SCHEMA_URL = "https://github.com/mozilla/mozilla-pipeline-schemas/archive/generated-schemas.tar.gz"
+
+
+@dataclass
+class LegacyPing:
+    name: str
+    versions: Sequence[str] = list
+
+    @property
+    def bigquery_fully_qualified_names(self) -> Sequence[str]:
+        table_names = []
+        for version in self.versions:
+            table_names.append(
+                f"moz-fx-data-shared-prod.telemetry_live."
+                + self.name.replace("-", "_")
+                + "_v"
+                + version.split(".")[1]
+            )
+        return table_names
```

Comment:
nit: We could use f-strings here, something like:
```suggestion
        for version in self.versions:
            table_name =  f"{self.name.replace("-", "_")}_v{version.split(".")[1]}"
            table_names.append(f"moz-fx-data-shared-prod.telemetry_live.{table_name})
        return table_names
```